### PR TITLE
Clarify README.md

### PR DESCRIPTION
**Pull request by lucyleeow**
I know this is WIP, but thought we could expand the description to provide more info. Might have been a bit confusing, especially to newcomers. Happy to change wording.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 [![tests](https://github.com/napari/napari-plugin-manager/workflows/test_and_deploy/badge.svg)](https://github.com/napari/napari-plugin-manager/actions)
 [![codecov](https://codecov.io/gh/napari/napari-plugin-manager/branch/main/graph/badge.svg)](https://codecov.io/gh/napari/napari-plugin-manager)
 
-A plugin that adds a plugin manager to [napari].
+[napari] plugin manager to enable installing [napari] packages, that is itself a plugin.
```
Comment (psobolewskiPhD) [inline README.md:11]:
```suggestion
[napari] plugin manager to enable installing [napari] plugins, that is itself a plugin.
```
Not sure we use `package` in this context, so maybe best to keep it simple and use plugin?

**Review by psobolewskiPhD — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lucyleeow) [reply README.md]: Good point, don't know what I was thinking! Thanks!

**Comment by lucyleeow** [comment]
Ah was using the github GUI and did not realise it creates a new branch instead of a fork. Will close this and do it properly, sorry @psobolewskiPhD !

**Comment by lucyleeow** [comment]
Opened here: https://github.com/napari/napari-plugin-manager/pull/18

**Pull request closed without merging**